routing/http/contentrouter/contentrouter_test.go

```diff
@@ -137,7 +171,117 @@ func TestFindProvidersAsync(t *testing.T) {
 	expected := []peer.AddrInfo{
 		{ID: p1},
 		{ID: p2},
+		{ID: p3},
 	}
 
 	require.Equal(t, expected, actualAIs)
 }
+
+func TestFindPeer(t *testing.T) {
+	ctx := context.Background()
+	client := &mockClient{}
+	crc := NewContentRoutingClient(client)
+
+	p1 := peer.ID("peer1")
+	ais := []types.Record{
+		&types.UnknownRecord{
+			Schema: "Unknown",
```

Comment:
```suggestion
			Schema: "unknown",
```

Reply:
I changed it but it doesn't matter as anything other than bitswap has always been ignored: https://github.com/ipfs/boxo/pull/422#discussion_r1305340163